tests/classdefs.pyi

```diff
@@ -0,0 +1,50 @@
+from abc import abstractmethod
+
+class Bad:
+    def __repr__(self) -> str:  # Y029 Defining __repr__ or __str__ in a stub is almost always redundant
+        ...
+
+    def __str__(self) -> str:  # Y029 Defining __repr__ or __str__ in a stub is almost always redundant
+        ...
+
+
+class AlsoBad:
+    def __repr__(self, /) -> str:  # Y029 Defining __repr__ or __str__ in a stub is almost always redundant
```

Comment:
This is making the test suite fail on 3.7. We could add some mechanism to the test runner for running tests only on some versions, but I'd be OK with just removing this test.

Reply:
Yeah, I know — I was waiting for Akuli's input :) (https://github.com/PyCQA/flake8-pyi/pull/109#issuecomment-1017468854).

I'd rather not leave this invariant untested, if possible — but happy to delete it, if it would make it horribly complex to keep it in.